src/index.js

```diff
@@ -28,53 +31,74 @@ addInput.addEventListener('keypress', (event) => {
   }
 });
 
+clearAll.addEventListener('click', async () => {
+  const removeCompleted = arr.filter((arr) => arr.completed === false);
+  arr = removeCompleted;
+  defaultExport();
+});
+
 export function component() {
-  arr.forEach((card, n) => {
-    const element = document.createElement('li');
-    element.setAttribute('id', `i${card.index}`);
-    element.innerHTML += `
-      <input type="checkbox" name="task${card.index}">
+  if (Array.isArray(arr) && arr.length > 0) {
+    arr.forEach(async (card, n) => {
+      const element = document.createElement('li');
+      element.setAttribute('id', `i${card.index}`);
+      element.innerHTML += `
```

Comment:
- [ OPTIONAL ] I think it would be great if you update the indexes as well since after deleting one element so the indexes it should be updated ✅✅ to be just 1, 2 , 3 if we deleted task 2 the indexes it would be 1 2 and so on 💯✨🔥